tensorflow/python/estimator/inputs/queues/feeding_functions.py

```diff
@@ -23,6 +23,7 @@
 import types as tp
 import numpy as np
 import six
+from six.moves import zip_longest
```

Comment:
Can you access this below as six.moves.zip_longest instead of adding the import here? We try to only import full modules.